winterfell/utils/formProperties.js

```diff
@@ -0,0 +1,54 @@
+function FormProperties(app) {
+    this.app = app;
+}
+
+module.exports = FormProperties;
+
+module.exports.FORM_ID = {
+  INTENT_TO_FILE_CLAIM: 'vba_21_0966_are'
```

Comment:
This type of form configuration information should be a static config file, so it is accessible to non-javascript code